src/mmw/js/src/analyze/models.js

```diff
@@ -436,11 +436,31 @@ function createAnalyzeTaskGroupCollection(aoi, wkaoi) {
             name: "pointsource",
             displayName: "Pt Sources",
             tasks: [
+                {
+                    name: "drb_pointsource",
+                    displayName: "Delaware River Basin Point Source Data (NPDES)",
+                    area_of_interest: aoi,
+                    wkaoi: wkaoi,
+                    taskName: "analyze/pointsource/drb",
+                    conusOnly: true
+                },
+                {
+                    name: "pa_pointsource",
+                    displayName: "Pennsylvania Point Source Data (NPDES)",
+                    area_of_interest: aoi,
+                    wkaoi: wkaoi,
+                    taskName: "analyze/pointsource/pa",
+                    conusOnly: true,
+                    lazy: true
+                },
                 {
                     name: "pointsource",
+                    displayName: "Continental US Point Source Data (NPDES)",
```

Comment:
I added these display names but let me know if there's specific text I should use for these labels instead